AlpsSDK/RemoteNotificationManager.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  RemoteNotificationManager.swift
+//  AlpsSDK
+//
+//  Created by Wen on 07.11.17.
+//  Copyright © 2017 Alps. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import UserNotifications
+
+protocol RemoteNotificationManagerDelegate: class {
+    func remoteNotificationManager(registerDeviceToken: Data)
```

Comment:
this is not delegate method. It's more of a facade for `AlpsManager` to deliver data to private `remoteNotificationManager`.

Note you don't use `delegate` anywhere.

Reply:
Yes I did that with the intention to not let the developer use RemoteNotificationManager class. But maybe it is better to make it public.